styles/sass/pages/_contact-us.scss

```diff
@@ -141,12 +146,18 @@ a{
   }
 }
 
+@media (min-width: 550px) {
+  .get-in-touch__map--cropped img {
+    margin: 0px 0px 0px 0px;
+  }
+}
+
```

Comment:
the best practice on SASS use the media queries within the selector like this:
```
.get-in-touch__map--cropped img {
    @media (min-width: 550px) {
       margin: 0px 0px 0px 0px;
  }
}
```
and 2 more observations:
1.- when you put 0 as the value you can skip the `px`:
`margin: 0 0 0 0;`
2.- if you will use the 0 value for all you could do something like this:
`margin: 0;`
https://www.w3schools.com/cssref/pr_margin.asp